maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/pom.xml

```diff
@@ -50,9 +50,16 @@
     <dependency>
       <groupId>org.eclipse.jgit</groupId>
       <artifactId>org.eclipse.jgit</artifactId>
-      <version>4.5.4.201711221230-r</version>
+      <version>${org.eclipse.jgit.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.eclipse.jgit</groupId>
+      <artifactId>org.eclipse.jgit.ssh.jsch</artifactId>
+      <version>${org.eclipse.jgit.version}</version>
```

Comment:
Why is this necessary? JGit has moved to MINA.

Reply:
Just try to maintain and upgrade an old dependency. Is there any problem with apache MINA?

Reply:
No problem with MINA, as far as I understood the JSch approach is dead.